### PR TITLE
New design on projects page

### PR DESCRIPTION
**Pull request by stefanlindbohm**
Added a few tweaks to paddings, otherwise straightforward.

One of the fixes broke the climate tips boxes on the homepage, but those were actually broken before, related to the fix itself. Change in this PR is to do gutters with paddings instead of margins, because otherwise widths are not properly accounted for (margins are not counted as part of width, but paddings are). Those tips boxes were broken in mobile screens, now broken in all breakpoints. So we'll have to fix those in a separate PR.

@emmabckstrm I'm happy to rebase after #329 is merged with the check list changes.

---

Preview: https://goclimateneutral-stagin-pr-330.herokuapp.com/climate-projects?new_design

### CODE REVIEW

File: app/views/projects/index_new.html.erb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  <ul class="list-checkmarks space-y-3 t:w-1/2">
+    <li><%=t 'views.projects.how.1' %></li>
+    <li><%=t 'views.projects.how.2' %></li>
+    <li><%=t 'views.projects.how.3' %></li>
+    <li><%=t 'views.projects.how.4' %></li>
+    <li><%=t 'views.projects.how.5' %></li>
+  </ul>
+
+  <div class="mt-12 text-center">
+    <%= link_to 'Read more about how our projects work', 'https://www.goclimate.com/se/blog/vad-ar-ett-klimatprojekt-egentligen/', class: 'button' %>
```
Comment (stefanlindbohm) [inline app/views/projects/index_new.html.erb:41]: Missing locale key

File: app/assets/main/components/toggler.scss [hunk 2]
```diff
@@ -15,4 +15,14 @@
...
+
+  &:checked ~ .toggler-checked\:block,
+  &:checked ~ * .toggler-checked\:block {
+    @apply block;
+  }
+
+  &:checked ~ .toggler-checked\:rotate-180,
+  &:checked ~ * .toggler-checked\:rotate-180 {
+    @apply rotate-180;
+  }
```
Comment (emmabckstrm) [inline app/assets/main/components/toggler.scss:27]: nice 😎

Comment (emmabckstrm) [reply app/assets/main/components/toggler.scss:27]: this makes me think that the component `collapse` might be obsolete? this way perhaps is easier?

**Comment by emmabckstrm** [comment]
forgot to say: it's looking soooooo good!

File: app/views/projects/index_new.html.erb [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      </div>
+    <% end %>
+  </div>
+
+  <div class="mt-12 text-center">
+    <a href="#all-projects" class="button" data-controller="scroll-to" data-action="scroll-to#scroll"><%=t 'views.projects.see_all' %></a>
+  </div>
+</section>
+
+<section class="section-padding">
```
Comment (kallenilver) [inline app/views/projects/index_new.html.erb:29]: Shouldn't this be centered? Looks a bit empty on the right side. Or are we waiting for an illustration?

File: config/locales/views/projects.en.yml [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
   views:
     projects:
+      heading: Our Climate Projects
+      latest_projects: Our latest projects
+      see_all: See all our projects
+      how:
+        heading: How our projects work
+        1: The purpose of all our projects is to reduce greenhouse gas emissions, for example by contributing to the transition towards renewable energy.
+        2: Each operational year of the projects, the avoided emissions are measured in carbon dioxide equivalents, which we purchase as carbon credits. This means that the impact can't later be undone.
+        3: We always pick the projects that make the most climate impact. This can, for example, be projects in low-income countries where the financial contribution makes a bigger difference.
```
Comment (kallenilver) [inline config/locales/views/projects.en.yml:11]: "be projects in low-income countries where the financial contribution makes a bigger difference." -> "be projects in highly fossil-fuel dependent countries where clean renewable energy makes a big difference by replacing coal power plants"

File: app/assets/main/components/toggler.scss (hunk 2 above)
Comment (stefanlindbohm) [reply app/assets/main/components/toggler.scss:27]: Yeah, I think so? I really like this idea that you read the markup and get the full picture of what's happening.

But this toggler thing might get out of hand too so maybe that's something to put on a technical improvements list to make a Tailwind plugin of or something?

File: config/locales/views/projects.en.yml [hunk 5]
```diff
@@ -1,6 +1,17 @@
...
+        4: In addition to the climate impact, the projects contribute to local sustainable development, for example by offering jobs or improving air quality.
+        5: Projects are certified by strict standards, such as the Gold Standard, and are regularly reviewed by third parties.
```
Comment (kallenilver) [inline config/locales/views/projects.en.yml:13]: "reviewed by third parties." -> "reviewed by independent third parties."

File: app/assets/main/components/toggler.scss (hunk 2 above)
Comment (emmabckstrm) [reply app/assets/main/components/toggler.scss:27]: Sounds good!

File: app/views/projects/index_new.html.erb (hunk 3 above)
Comment (stefanlindbohm) [reply app/views/projects/index_new.html.erb:29]: Waiting for illustration

**Pull request merged**